fr trans

d0ob opened the pull request:
Well, today in the "how github played tricks on me" book, I learned that half of the time I was sending pull request to my own repository xD

d0ob committed:
FR transl

I only see auto filter remaining, it will not be done today, much more talkative ! 
And since I don't use it myself, I vant to play abit with it to be sure I don't missunderstood anything !

d0ob committed:
Merge pull request #1 from d0ob/d0ob-patch-1

FR transl

Wyrrrd commented:
You refer to rocket turrets once as "tourelles lance-roquettes" and everywhere else as "tourelles lance-missiles". Could you update this PR again for consistency?

Wyrrrd commented:
Also, filename fixing needed, `strings.cfg` instead of `locale.cfg`.

d0ob committed:
Correction

Indeed, my bad... Should have definitely re-re-read.
I saw the [rocketturrets] wasn't translated in other files so I didn't either.

For the name thought, Rocket_turret's local files don't seem to be called string.cfg but locale.cfg

Wyrrrd commented:
My bad, looks like the locale file can be named anything.cfg
